man/systemd.network.xml

```diff
@@ -1890,6 +1890,20 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SocketPriority=</varname></term>
+        <listitem>
+          <para>The linux socket option <constant>SO_PRIORITY</constant> applied to the raw IP socket used for
+          initial DHCPv4 messages. Defaults to 0. Usual values ranges from 0 to 6.
+          More details about <constant>SO_PRIORITY</constant> socket option in
+          <citerefentry><refentrytitle>socket</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+          Can be used in conjunction with netdev VLAN EgressQOSMap to set the 802.1Q VLAN ethernet
```

Comment:
`<varname>EgressQosMaps=</varname>`
As also said in the above, please clarify the section name `[VLAN]`.